python/collect_emotiv_data.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+from lib.cortex import Cortex
```

Comment:
when i used from  lib.cortex import cortex , an error message "no module called lib could be fined" 
how can i fix it? 
thanks